omod/src/main/java/org/openmrs/module/cohort/web/resource/CohortMemberResource.java

```diff
@@ -134,12 +134,7 @@ public CohortMember getByUniqueId(String uuid) {
 	
 	@Override
 	public void purge(CohortMember cohortMember, RequestContext context) throws ResponseException {
-		boolean purge = Boolean.getBoolean(context.getParameter("purge"));
-		if (purge) {
 			Context.getService(CohortMemberService.class).purgeCohortMember(cohortMember);
```

Comment:
Should also be formatted so this is no longer indented.

Reply:
Oh sure @ibacher . Though after testing, manually with postman & the legacy UI Webservices Test page for REST URIs, its returns a "success" response but still does the normal delete and the cohort member remains in the database. Have tried tracing the code flow and can't seem to figure out why it still isn't purging.

Reply:
So that seems to mean either:

1. `purge()` isn't getting called
2. `purgeCohortMember()` isn't calling `delete()`

Can you show what you tested in Postman?

Reply:
![Screenshot (57)](https://github.com/openmrs/openmrs-module-cohort/assets/19221635/dc7a4ebd-c59d-4ca7-a332-74722ccf5c28)

http://localhost:8888/openmrs/ws/rest/v1/cohortm/cohortmember/519fb823-bd7e-4360-b41c-d90cf91a254b?purge=true

Reply:
@WodPachua Sorry, I missed this. When you do that via Postman, is the member deleted?

Reply:
@ibacher Yes, the delete is successful, at least the database shows so.

Reply:
Ok, so I think unindent the code, so that it's formatted correctly and we can merge this in.

Reply:
Alright @ibacher , I have fixed that.